### DISCUSSION TITLE
item assignment

### DISCUSSION THREAD

**Discussion by pure-rgb** [jax-ml/jax#11035] [General] [answered]
In numpy, 

```python
a[:, :, 0] += self.b[0] - 1
a[:, :, 1] *= 2 * self.b[1] - 1
```

how to do this in jax?

**Comment by jakevdp** [comment, marked as answer]
This is covered at https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#in-place-updates. Please let me know if that answers your question!
